test/testUtils/mtohUtils.py

```diff
@@ -88,7 +88,7 @@ def setUp(self):
         # Maya is not closed/reset between each test of a test suite,
         # so open a new file before each test to minimize leftovers
         # from previous tests.
-        cmds.file(new=True, force=True)
+        mayaUtils.openNewScene()
```

Comment:
Also with regards to @lilike-adsk's comment on providing a way to override certain default settings or using a skip flag, I think a simpler way is that if a test really needs specific settings, it can override `setUp` method and set whatever settings it needs, which means we don't have to implement a specific mechanism for this.